### ISSUE TITLE
Promise was rejected with a non-error

### ISSUE THREAD

**Issue by Tom910** [bramstein/fontfaceobserver#81]
I get an error when using the fontfaceobserver and bluebird

```
bluebird.js:1482 Warning: a promise was rejected with a non-error: [object Undefined]
    at warn (webpack:///./~/bluebird/js/browser/bluebird.js?:1081:19)
    at Promise._warn (webpack:///./~/bluebird/js/browser/bluebird.js?:762:12)
    at Promise._rejectCallback (webpack:///./~/bluebird/js/browser/bluebird.js?:3104:14)
    at eval (webpack:///./~/bluebird/js/browser/bluebird.js?:3118:17)
```
fontfaceobserver@2.0.4 and bluebird@3.4.1

code:
```
var fontA = new FontFaceObserver('Family A');
var fontB = new FontFaceObserver('Family B');

Promise.all([fontA.load(), fontB.load()]).then(function () {
  console.log('Family A & B have loaded');
});
```

**Comment by themoonrat** [comment]
This pending PR fixes it: https://github.com/bramstein/fontfaceobserver/pull/76

**Comment by bramstein** [comment]
This should be fixed in 2.0.5, thanks to @themoonrat.
